.yamato/all-hdrp.yml

```diff
@@ -12,21 +12,21 @@ PR_HDRP_trunk:
     dependencies:
       -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_Standalone_mono_Linear_trunk
          rerun: on-new-revision
-      -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_playmode_mono_Linear_trunk
+      -  path: .yamato/hdrp-win-dx11.yml#HDRP_Win_DX11_playmode_cache_mono_Linear_trunk
```

Comment:
I wonder if 
- keeping the same name (just `playmode`) for the playmode cache jobs and 
- renaming the regular playmode jobs to `playmode_no_cache` 

would make sense? That would allow us to keep the same unique names in yamato, and keep the run history. 
Maybe that's too much rework for too little benefits though, only an idea.

Reply:
this is a good point. my thought is that "no cache" has been the default for so long that i dont want to sneak in like a "hidden" change, but maybe that doesnt matter 🤔